demo/client/backend/mock/README.md

```diff
@@ -0,0 +1,35 @@
+# Start with go chaincode
+
+    $ go run .
+
+# Start with fpc chaincode
+
+Note that you need to start the docker container from the fpc root
+folder
+
+    $ docker run -p 3000:3000 -it -v $PWD:/project/src/github.com/hyperledger-labs/fabric-private-chaincode hyperledger/fabric-private-chaincode-dev
+    $ make clean && make
+    $ cd ecc
+    $ make sym-links
+
+We need some more go deps here
+
+    $ go get github.com/golang/protobuf/proto && go get golang.org/x/sync/semaphore
+    $ go get github.com/gin-contrib/cors && go get github.com/gin-gonic/gin
+
+Next build your FPC chaincode.
```

Comment:
FYI: for the existing chaincodes in the repo, only echo will work until i fix creator, auction_test and helloworld will both fail ...

Reply:
got stuck on this for a little while, cause I was trying helloworld, thanks!